shell/assets/translations/en-us.yaml

```diff
@@ -1769,10 +1769,13 @@ cluster:
       harvester: '{tag}'
   providerGroup:
     create-custom1: Use existing nodes and create a cluster using RKE
-    create-custom2: Use existing nodes and create a cluster using RKE2/K3s
+    # Added by Verrazzano Start
+    # create-custom2: Use existing nodes and create a cluster using RKE2/K3s
+    create-custom2:
+    # Added by Verrazzano End
```

Comment:
Just a nit:  since the two Verrazzano blocks are next to each other, you don't need to START, and END twice (one wrapper around both would have been sufficient).

Reply:
fixed